tools/python/upload_and_run_browserstack_tests.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import time
+
+import requests
+
+parser = argparse.ArgumentParser("Upload and run BrowserStack tests")
```

Comment:
we usually want to put the script logic to execute inside the check `if __name__ == '__main__'`. then, it only gets executed when the script is called directly and not in other contexts, like when the script is imported by something else.

more details here: https://docs.python.org/3/library/__main__.html